ArduCopter/mode_follow.cpp

```diff
@@ -103,6 +104,7 @@ void Copter::ModeFollow::run()
 
         // slow down horizontally as we approach target (use 1/2 of maximum deceleration for gentle slow down)
         const float dist_to_target_xy = Vector2f(dist_vec_offs_neu.x, dist_vec_offs_neu.y).length();
+
         copter.avoid.limit_velocity(pos_control->get_pos_xy_p().kP().get(), pos_control->get_accel_xy() * 0.5f, desired_velocity_xy_cms, dir_to_target_xy, dist_to_target_xy, copter.G_Dt);
 
         // limit the horizontal velocity to prevent fence violations
```

Comment:
@rmackay9 Why do the other two copter.avoid.*() function calls, a couple of lines above and a couple of lines bellow use `copter.G_Dt` and the function call in the next line uses just `G_Dt` ?